lib/prompts/projectDevTargetAccountPrompt.ts

```diff
@@ -99,22 +118,24 @@ const selectSandboxTargetAccountPrompt = async (
     'sandbox account',
     choices
   );
-};
+}
 
-const selectDeveloperTestTargetAccountPrompt = async (
-  accounts,
-  defaultAccountConfig
-) => {
+export async function selectDeveloperTestTargetAccountPrompt(
+  accounts: CLIAccount[],
+  defaultAccountConfig: CLIAccount
+): Promise<DeveloperTestAccount | CLIAccount> {
   const defaultAccountId = getAccountId(defaultAccountConfig.name);
-  let devTestAccountsResponse = undefined;
+  let devTestAccountsResponse: FetchDeveloperTestAccountsResponse | undefined;
   try {
-    const { data } = await fetchDeveloperTestAccounts(defaultAccountId);
-    devTestAccountsResponse = data;
+    if (defaultAccountId) {
```

Comment:
I should have caught this before, but this appears to be another subtle behavior change. We should throw if there's no defaultAccountId

Reply:
Addressed in https://github.com/HubSpot/hubspot-cli/pull/1293/commits/ad7654ea3612b8f2294af6d5063d4e2360a2c5ec